components/finops/logicapp.tf

```diff
@@ -19,3 +19,24 @@ resource "azurerm_user_assigned_identity" "finopslogicapp-mi" {
   name                = "finopslogicapp-${var.env}-mi"
   tags                = module.ctags.common_tags
 }
+
+data "local_file" "logic_app" {
+  count    = var.env == "ptl" ? 1 : 0
+  filename = "${path.module}/workflow-${var.env}.json"
+}
+
+resource "azurerm_resource_group_template_deployment" "logic_app_deployment" {
+  count               = var.env == "ptl" ? 1 : 0
+  resource_group_name = azurerm_resource_group.finopsrg.name
+  deployment_mode     = "Incremental"
+  name                = "logic-app-deployment"
+
+  template_content = data.local_file.logic_app[0].content
+
+  parameters_content = jsonencode({
+    "logic_app_name" = { value = azurerm_logic_app_workflow.finopslogicapp[0].name }
+    "location"       = { value = azurerm_logic_app_workflow.finopslogicapp[0].location }
+  })
+
+  depends_on = [azurerm_logic_app_workflow.finopslogicapp[0]]
```

Comment:
...explicitly needed? intrinsic dependency may already take care of this on line 37 & 38

Reply:
Just cautious :p